### PR TITLE
nodemon: add page

### PR DESCRIPTION
**Pull request by Scaledi**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/nodemon.md
Comment (navarroaxel) [inline pages/common/nodemon.md:14]:
```suggestion
- Ignore specific files:
```

File: pages/common/nodemon.md
Comment (navarroaxel) [inline pages/common/nodemon.md:4]: I think we should use the official website:
```suggestion
> More information: <https://nodemon.io>.
```
or, the official docs site:
```suggestion
> More information: <https://github.com/remy/nodemon#nodemon>.
```

File: pages/common/nodemon.md
Comment (navarroaxel) [inline pages/common/nodemon.md:20]:
```suggestion
`nodemon {{path/to/file.js}} {{arguments}}`
```

File: pages/common/nodemon.md
Comment (navarroaxel) [inline pages/common/nodemon.md:18]:
```suggestion
- Pass arguments to your node application:
```

File: pages/common/nodemon.md
Comment (navarroaxel) [inline pages/common/nodemon.md:24]: We try to set template to commands instead a specific one.
```suggestion
`nodemon --exec "{{command}}" {{path/to/script_file}}`
```

File: pages/common/nodemon.md
Comment (navarroaxel) [inline pages/common/nodemon.md:22]:
```suggestion
- Running non-node scripts:
```

File: pages/common/nodemon.md
Comment (navarroaxel) [inline pages/common/nodemon.md:16]:
```suggestion
`nodemon --ignore {{path/to/file_or_directory}}`
```

File: pages/common/nodemon.md
Comment (navarroaxel) [inline pages/common/nodemon.md:6]:
```suggestion
- Watch a spesific file for reloads:
```

Comment (Scaledi) [reply pages/common/nodemon.md:6]: Dont see any changes here

Comment (navarroaxel) [reply pages/common/nodemon.md:6]: updated! sorry.

File: pages/common/nodemon.md
Comment (navarroaxel) [inline pages/common/nodemon.md:22]: Sometimes I forgot the rules too:
```suggestion
- Run non-node scripts:
```

**Comment by Scaledi** [comment]
@navarroaxel should all be fixed

**Comment by navarroaxel** [comment]
> @navarroaxel should all be fixed

Thanks @Scaledi ! we should wait the feedback from another reviewer.

**Comment by Scaledi** [comment]
> > @navarroaxel should all be fixed
> 
> Thanks @Scaledi ! we should wait the feedback from another reviewer.

cool. any reason why?
just trying to figure out how this works.... don't mean that in a rude way! just tryen to learn.

**Comment by navarroaxel** [comment]
> > > @navarroaxel should all be fixed
> > 
> > 
> > Thanks @Scaledi ! we should wait the feedback from another reviewer.
> 
> cool. any reason why?
> just trying to figure out how this works.... don't mean that in a rude way! just tryen to learn.

np, we have a [maintainer's guide](https://github.com/tldr-pages/tldr/blob/master/contributing-guides/maintainers-guide.md) and the rule is `2 approved reviews by maintainers` before merge. Also I'm from :argentina: and English is not my mother language. So, I could be wrong about grammar.

**Comment by Scaledi** [comment]
> > > > @navarroaxel should all be fixed
> > > 
> > > 
> > > Thanks @Scaledi ! we should wait the feedback from another reviewer.
> > 
> > 
> > cool. any reason why?
> > just trying to figure out how this works.... don't mean that in a rude way! just tryen to learn.
> 
> np, we have a [maintainer's guide](https://github.com/tldr-pages/tldr/blob/master/contributing-guides/maintainers-guide.md) and the rule is `2 approved reviews by maintainers` before merge. Also I'm from 🇦🇷 and English is not my mother language. So, I could be wrong about grammar.

see my edit... just got to the page before you reviewed it!

**Review by sbrl — CHANGES_REQUESTED** [review]
Thanks for the new page, @Scaledi! I've left some comments below for you to review.

Welcome to tldr pages :slightly_smiling_face:

File: pages/common/nodemon.md
Comment (navarroaxel) [reply pages/common/nodemon.md:8]: Just 1 slash:
```suggestion
`nodemon --inspect {{path/to/file.js}}`
```

**Comment by Scaledi** [comment]
> Thanks for the new page, @Scaledi! I've left some comments below for you to review.
> 
> Welcome to tldr pages 🙂

Finished it all :) hope its better now!

**Review by sbrl — APPROVED** [review]
Thanks, @Scaledi :smiley_cat:

**Pull request merged**
